contents/messages.tsx

```diff
@@ -0,0 +1,227 @@
+import { startRegistration } from "@simplewebauthn/browser"
+import {
+  convertCOSEtoPKCS,
+  decodeAttestationObject,
+  decodeClientDataJSON,
+  generateChallenge,
+  isoBase64URL,
+  isoUint8Array,
+  parseAuthenticatorData
+} from "@simplewebauthn/server/helpers"
+import type {
+  AttestationConveyancePreference,
+  AuthenticatorAttachment,
+  COSEAlgorithmIdentifier,
+  PublicKeyCredentialCreationOptionsJSON,
+  PublicKeyCredentialDescriptorJSON,
+  PublicKeyCredentialType,
+  UserVerificationRequirement
+} from "@simplewebauthn/typescript-types"
+import { AbiCoder } from "ethers"
+import type { PlasmoCSConfig } from "plasmo"
+import { useState } from "react"
+
+import { useMessage } from "@plasmohq/messaging/hook"
+
+const ethersAbi = AbiCoder.defaultAbiCoder()
+export const config: PlasmoCSConfig = {
+  matches: ["<all_urls>"],
+  // The "all_frames" field determines whether files are injected into all frames meeting URL requirements or only into the top frame in a tab.
+  all_frames: true,
+  // Specifies the timing for injecting JavaScript files into the web page. The default is document_idle, injecting a script into a target context to run at document_idle. If the script returns a promise, the browser waits for it to settle before returning the result.
+  run_at: "document_idle"
+}
+
+export enum ContentMethod {
+  content_createWebauthn = "content_createWebauthn"
+}
+
+export type ContentRequestArguments = ContentCreateWebauthnArguments
+
+export type ContentCreateWebauthnArguments = {
+  method: ContentMethod.content_createWebauthn
+  params: {
+    tabId?: number
+    user?: string
+    chalBase64Url?: string
+    authAttach?: AuthenticatorAttachment
+  }
+}
+
+const Messages = () => {
+  const [status, setStatus] = useState<string>("")
+
+  const { data: _ } = useMessage<Record<string, any>, Record<string, any>>(
```

Comment:
Q: Does plasmo provide other method for content script listening to background message? `useMessage` would restrict this function in only react component, but we might not render any component in the content script.

Reply:
I seem to lack a clear understanding of the execution details of this useMessage...
Perhaps we can clarify this further in an online discussion.